definitions/checks/unread_mail.rb

```diff
@@ -0,0 +1,40 @@
+class Checks::UnreadMail < ForemanMaintain::Check
+  metadata do
+    label :unread_mail
+    description 'Check for unread mail for root user.'
+    tags :default
+  end
+
+  attr_reader :file_path
+
+  def initialize
+    @file_path = '/var/spool/mail/root'
```

Comment:
Would it be feasible to retrieve mail spool path from site-specific configuration file `/etc/login.defs`? This file defines the default path for `MAIL_DIR`, and its value differs based on distro.

On Fedora*
```shell
[anurag@coruscant ~]$ cat /etc/redhat-release 
Fedora release 25 (Twenty Five)
[anurag@coruscant ~]$ grep ^MAIL_DIR /etc/login.defs 
MAIL_DIR	/var/spool/mail
[anurag@coruscant ~]$ 
```

On Debian*
```shell
anurag@ellen:~$ cat /etc/debian_version 
wheezy/sid
anurag@ellen:~$ grep ^MAIL_DIR /etc/login.defs 
MAIL_DIR        /var/mail
anurag@ellen:~$ 
```
You may want to retain the default value of `/var/spool/mail/root` and fallback to it if the above queries fail.